interface/billing/billing_report.php

```diff
@@ -98,1137 +100,1217 @@
-        <?php } ?>
-    </ul>
-</span>
-<span class="btn-group pull-left">
-    <button type="button" class="subbtn dropdown-toggle" data-toggle="dropdown" name="bn_process_hcfa_support"
-        title="<?php echo xla('A claim must be selected to enable this menu.')?>"><?php echo xla('HCFA FORM')?>
-    <span class="caret"></span></button>
-    <ul class="dropdown-menu" role="menu">
-        <li><button type="submit" class="btn btn-link" name="bn_process_hcfa" onclick="MarkAsCleared(2)"
-            title="<?php echo xla('Generate and download CMS 1500 paper claims')?>"><?php echo xla('CMS 1500 PDF')?></button></li>
-        <?php if ($GLOBALS['preprinted_cms_1500']) { ?>
-        <li><button type="submit" class="btn btn-link" name="bn_process_hcfa_form" onclick="MarkAsCleared(2)"
-            title="<?php echo xla('Generate and download CMS 1500 paper claims on Preprinted form')?>"><?php echo xla('CMS 1500 incl FORM')?></button></li>
-        <?php } ?>
-        <li><button type="submit" class="btn btn-link" name="bn_hcfa_txt_file" onclick="MarkAsCleared(3)"
-            title="<?php echo xla('Making batch text files for uploading to Clearing House and will mark as billed')?>"><?php echo xla('CMS 1500 TEXT')?></button></li>
-    </ul>
-</span>
-<?php if ($GLOBALS['ub04_support']) { ?>
-<span class="btn-group pull-left">
-    <button type="button" class="subbtn dropdown-toggle" data-toggle="dropdown" name="bn_process_ub04_support"
-        title="<?php echo xla('A claim must be selected to enable this menu.')?>"><?php echo xla('UB04 FORM')?><span class="caret"></span></button>
-    <ul class="dropdown-menu" role="menu">
-        <li><button type="submit" class="btn btn-link" name="bn_process_ub04_form"
-            title="<?php echo xla('Generate and download UB-04 CMS1450 with form')?>"><?php echo xla('UB04 FORM PDF')?></button></li>
-        <li><button type="submit" class="btn btn-link" name="bn_process_ub04"
-            title="<?php echo xla('Generate and download UB-04 CMS1450')?>"><?php echo xla('UB04 TEXT PDF')?></button></li>
-    </ul>
-</span>
-    <?php } ?>
-<button type="submit" data-open-popup="true" class="subbtn-warning" name="bn_mark" title="<?php echo xla('Post to accounting and mark as billed')?>"><?php echo xla('Mark as Cleared')?></button>
-<button type="submit" data-open-popup="true" class="subbtn-warning" name="bn_reopen" title="<?php echo xla('Mark as not billed')?>"><?php echo xla('Re-Open')?></button>
-&nbsp;&nbsp;&nbsp;
-<?php echo xlt('CMS 1500 Margins'); ?>:
-<?php echo xlt('Left'); ?>:
-<input type='text' size='2' name='left_margin' value='<?php echo attr($left_margin); ?>'
-                title='<?php echo xla('HCFA left margin in points'); ?>' />
-<?php echo xlt('Top'); ?>:
-<input type='text' size='2' name='top_margin' value='<?php echo attr($top_margin); ?>'
-                title='<?php echo xla('HCFA top margin in points'); ?>' />
-<?php if ($ub04_support) { ?>
```

Comment:
1. It looks like the code block below is missing in new file.
2. The following prior changes have also been omitted in this file: https://github.com/zbig01/openemr/commit/f4099d206d9c3bf60e72fc41745f1beeb31e9c14#diff-815dba801e0d5ef540ecd6fcce88d63d

@zbig01 , it appears most of the changes from the prior changes are here, but both of these changes are not included. Note you can see he history of this scrip by clicking view above then click History. We are going to likely need to do this for each script and ensure no changes have been lost. The thing that is tough is that we do not know the starting point (ie. at what date are we safe in the past that no changes were removed).

Reply:
confirmed fixed